app/Iterators/BagItIterator.scala

```diff
@@ -20,7 +20,7 @@ class BagItIterator(pathToFolder : String, collection : Option[models.Collection
 
   private def addBagItTextToZip(pathToFolder : String, totalbytes: Long, totalFiles: Long, zip: ZipOutputStream, collection: models.Collection, user: Option[models.User]) = {
     zip.putNextEntry(new ZipEntry(pathToFolder + "/bagit.txt"))
-    val softwareLine = "Bag-Software-Agent: clowder.ncsa.illinois.edu\n"
+    val softwareLine = "Bag-Software-Agent: clowder\n"
```

Comment:
I couldn't find any specific information about what should go in this field. Did you find something regarding this? I saw something along the lines emails: `clowder <https://clowderframework.org/>`.

Reply:
the agent definition is typically the name of the software, along with versioning. For example, this is how a similar field is used in the HTTP standard:

https://en.wikipedia.org/wiki/User_agent

My reasoning here would be that at a minimum it should be clowder rather than a specific domain as a clowder instance can (and will) be installed in other locations than at NCSA